install-to-air-gapped.md

```diff
@@ -0,0 +1,115 @@
+# Installing Application Service Adapter to Air-Gapped Environments
+
+For instructions on how to install prerequisites in air-gapped environments, see
+* [Tanzu Application Platform](https://docs.vmware.com/en/Tanzu-Application-Platform/1.1/tap/GUID-install.html#relocate-images-to-a-registry-0)
+* [Tanzu Build Service](https://docs.vmware.com/en/Tanzu-Build-Service/1.5/vmware-tanzu-build-service/GUID-installing-no-kapp.html#installation-to-air-gapped-environment)
+
+Below topics describe how Application Service Adapter for VMware Tanzu Application Platform system can can be installed to a Kubernetes Cluster and registry that are air-gapped from external traffic.
+
+* [Relocate Images to a Registry](#relocate-images-to-registry)
+* [Install the package repository](#install-package-repo)
+
+----
+
+## <a id="relocate-images-to-registry"></a>Relocate Images to a Registry (Air-Gapped)
+
+This procedure relocates images from the Tanzu Network registry to an internal image registry that's available to the air-gapped environment via a local machine.
+The local machine should have access to the air-gapped environment.
+
+1. Log in to the Tanzu Network registry with your Tanzu Network credentials:
+   ```bash
+    docker login registry.tanzu.vmware.com
+   ```
+   
+2. Copy the Application Service Adapter bundle to a tarball with the [Carvel](https://carvel.dev/imgpkg/) `imgpkg` tool by running:
+   ```bash
+    imgpkg copy -b registry.tanzu.vmware.com/app-service-adapter/tas-adapter-package-repo:0.6.1 --to-tar tas-adapter-package-repo.tar
```

Comment:
Meta question: I wonder how we'll keep these versions updated over time. As we cut patch versions and stuff. It would be nice if there was a way it could be templated, but not sure what this docs framework provides. 🤔

Reply:
I also wonder if for these air gapped docs we should tell them to export a `$TAS_ADAPTER_VERSION` variable or something and tell them to set it to the version they're installing. Then these particular docs could be more evergeen. Not sure what strategy TAP docs take for this. Thoughts @a-peek4 @emalm ?

Reply:
@tcdowney What I've seen for air gapped support has only been on a component level - no TAP wide docs. So that may either be the plan or it's too early before the release to see those docs materialize yet.

We can certainly use variables, that's what we started doing in TAP to make it easier to update the version numbers each release so there was less manual doc work. Take a look at it here: https://docs-staging.vmware.com/en/draft/Tanzu-Application-Platform/1.2/tap/GUID-install.html
VERSION-NUMBER is your Tanzu Application Platform version. For example, 1.2.0.